README.md

```diff
@@ -12,6 +12,7 @@ This is a Node.js client for [Jenkins](http://jenkins-ci.org/).
 - plugin: [list](#plugin-list)
 - queue: [list](#queue-list), [item](#queue-item), [cancel](#queue-cancel)
 - view: [get config](#view-config-get), [set config](#view-config-set), [create](#view-create), [destroy](#view-destroy), [exists](#view-exists), [get](#view-get), [list](#view-list), [add job](#view-add), [remove job](#view-remove)
+- credentials: [create](#credential-create), [exists](#credential-exists), [get config](#credential-get-config), [set config](#credential-set-config), [destroy](#credential-destroy), [list](#credential-list)
```

Comment:
Can we move this below build (basically everything after the `jenkins` section is sorted alphabetically) and the sections below to between `jenkins.build.*` and `jenkins.job.*`.

I'd also change `#credential-*` to `#credentials-*` for all the anchors/links.